### PR TITLE
SB-9: Add Firebase and Credentials

### PR DESCRIPTION
**Pull request by EzeLamar**
Changes:
* Add Firebase library
* Update README with the steps to attach the credentials of firebase Project.
* Ommit .env file

### CODE REVIEW

**Comment by EzeLamar** [comment]
[Add Firebase API & creds to Project](https://trello.com/c/X8xiQlBe/9-add-firebase-api-creds-to-project)

File: README.md [hunk 1]
```diff
@@ -34,3 +34,16 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
...
+# Firebase
+It is necessary to create the file .env in the root directory with the credentials of the corresponding Firebase Project with the next structure:
+```
+NEXT_PUBLIC_FIREBASE_API_KEY=api-key
+NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=auth-domain
+NEXT_PUBLIC_FIREBASE_PROJECT_ID=project-id
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=storage-bucket
+NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=sender-id
+NEXT_PUBLIC_FIREBASE_APP_ID=app-id
+NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID=analytic-id
```
Comment (lubualo) [inline README.md:48]:
```suggestion
NEXT_PUBLIC_FIREBASE_API_KEY=<api-key>
NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN=<auth-domain>
NEXT_PUBLIC_FIREBASE_PROJECT_ID=<project-id>
NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=<storage-bucket>
NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=<sender-id>
NEXT_PUBLIC_FIREBASE_APP_ID=<app-id>
NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID=<analytic-id>
```

Comment (lubualo) [reply README.md:48]: I think this makes more explicit that it is something that one has to modify.

**Review by lubualo — APPROVED** [review]
The code seems right to me.

**Pull request merged**